os/kernel/binary_manager/binary_manager_load.c

```diff
@@ -257,6 +204,32 @@ int binary_manager_load_binary(int bin_idx)
 	return ERROR;
 }
 
+/****************************************************************************
+ * Name: binary_manager_load
+ *
+ * Description:
+ *	 This function loads user binary with binary index.
+ *
+ ****************************************************************************/
+static int binary_manager_load(int bin_idx)
```

Comment:
This is just wrapper of `binary_manager_load_fs_binary`. I think this is not needed.